modules/sitemap/server/routes/[prefix]/sitemap.xml.ts

```diff
@@ -0,0 +1,23 @@
+import SitemapRenderer from '../../../SitemapRenderer.js'
```

Comment:
You should be able to import it via `@storefront-x/sitemap/SitemapRenderer.js` import.